.travis.yml

```diff
@@ -76,6 +76,10 @@ before_install:
       travis_terminate 0;
     fi;
   fi;
+- if [[ $TRAVIS_SECURE_ENV_VARS == true && $TEST_DOCKER == true ]]; then
+    echo "Logging in to dockerhub";
+    docker login --username=$DOCKER_SERVICE_LOGIN --password=$DOCKER_SERVICE_PASS;
```

Comment:
Have you confirmed that this command doesn't leak anything sensitive into the logs? Also, do we still need the separate login below for the cron task?

Reply:
It doesn't leak anything to logs although it does complain that using `--password` is a potential way to leak stuff.  I think it's ok for now.  

We do need the separate login below because that cron task doesn't have TEST_DOCKER  set to true.